scipy/stats/tests/test_distributions.py

```diff
@@ -1374,6 +1374,56 @@ def test_stats2(self):
         assert_equal((m, s), (0., 0.))
         assert_allclose((v, k), (4., 3.25))
 
+    @pytest.mark.parametrize("rvs_loc", [-5, 0, 1, 2])
+    @pytest.mark.parametrize("rvs_scale", [1, 2, 3, 10])
+    def test_fit(self, rvs_loc, rvs_scale):
+        # tests that various inputs follow expected behavior
+        # for a variety of `loc` and `scale`.
+        data = stats.laplace.rvs(size=100, loc=rvs_loc, scale=rvs_scale)
+
+        # MLE estimates are given by
+        loc_mle = np.median(data)
+        scale_mle = np.sum(np.abs(data - loc_mle)) / len(data)
+
+        # standard outputs should match MLE
+        loc, scale = stats.laplace.fit(data)
+        assert_equal(loc, loc_mle)
+        assert_equal(scale, scale_mle)
+
+        # fixed parameter matches MLE should use MLE for other
+        loc, scale = stats.laplace.fit(data, floc=loc_mle)
+        assert_equal(scale, scale_mle)
+        loc, scale = stats.laplace.fit(data, fscale=scale_mle)
+        assert_equal(loc, loc_mle)
+
+        # error raised when both `floc` and `fscale` are fixed
+        assert_raises(RuntimeError, stats.laplace.fit, data, floc=loc_mle,
+                      fscale=scale_mle)
+
+        # error is raised with non-finite values
+        assert_raises(RuntimeError, stats.laplace.fit, [np.nan])
+        assert_raises(RuntimeError, stats.laplace.fit, [np.inf])
+
+    @pytest.mark.parametrize("rvs_scale,rvs_loc", [(10, -5)])
+    def test_fit_MLE_comp_optimzer(self, rvs_loc, rvs_scale):
+        data = stats.laplace.rvs(size=1000, loc=rvs_loc, scale=rvs_scale)
+
+        # that MLE estimates are almost equal to optimizer
```

Comment:
Be more precise:
```suggestion
        # test that the analytical MLEs are close to the numerically optimized MLEs
```

Reply:
Of course this will need to change if you replace the assertions. The point is that using "MLE estimates" to refer to the _analytical estimates_ and "optimizer" to refer to the _numerically optimized estimates_ is ambiguous.

Reply:
I will be more specific with the language.